util/Migrator/DbScripts/2024-05-05_00_UpdateManyAuthRequests.sql

```diff
@@ -0,0 +1,61 @@
+IF NOT EXISTS (
+    SELECT
+        *
+    FROM
+        sys.types
+    WHERE
+        [Name] = 'AuthRequestType' AND
+        is_user_defined = 1
+)
+BEGIN
+    CREATE TYPE [dbo].[AuthRequestType] AS TABLE(
```

Comment:
In discussions with @withinfocus, we would like to move away from creating type objects in sql server  and use something a little more generic like json for bulk inserts

Reply:
Implemented as suggest on https://github.com/bitwarden/server/pull/4053/commits/cea0589d554ff1ec1a353450bda5c338c2459aec and then with some fixes on https://github.com/bitwarden/server/pull/4053/commits/632e3fa1c3d7b7a582c86c4bd90e520300e836e3 that are separated because I merged in between them.